opencog/pln/types/atom_types.script

```diff
@@ -7,3 +7,25 @@
 //
 // for a formal definition of its semantics.
 INTENSIONAL_DIFFERENCE_LINK <- ORDERED_LINK,EVALUATABLE_LINK
+
+// Temporal Reasoning
+
+// Warning: some of these links are already defined in the opencog
+// repository, see
+//
+// opencog/opencog/nlp/types/atom_types.script
+//
+// and the spacetime repository, see
+//
+// spacetime/opencog/spacetime/atom-types/atom_types.script.
+//
+// Also note that SequentialAnd and SequentialOr are already defined
+// in the atomspace repository.
+
+// For the definition of these links see
+// https://wiki.opencog.org/w/Category:Atom_Types
+AT_TIME_LINK <- ORDERED_LINK
+TIME_INTERVAL_LINK <- ORDERED_LINK
+TIME_NODE <- NODE
+PREDICTIVE_IMPLICATION_LINK <- ORDERED_LINK
+PREDICTIVE_IMPLICATION_SCOPE_LINK <- SCOPE_LINK
```

Comment:
@ngeiswei With the exception of `PREDICTIVE_IMPLICATION_SCOPE_LINK`  all links are defined in [here](https://github.com/opencog/spacetime/blob/b9cc4d85ad03c5642777d0b76fd2d2c18a064876/opencog/spacetime/atom-types/atom_types.script)

Reply:
Thanks @AmeBel, I'm aware of that but I have redefined them here until I get to start using spacetime.